### PR TITLE
Create table and answer queries

### PR DESCRIPTION
**Pull request by keebot-png**
## The following criteria has been completed in line with project requirements:
- [x] Create a data base called 'vet_clinic'.
- [x] Created a table called 'animals'.
- [x] Inserted the correct data in the table.
- [x] Answered the questions below and captured screenshots of the results.

### Solutions to the questions
<details>
<summary>Find all animals whose name ends in "mon"</summary>

</details>
<details>
<summary>List the name of all animals born between 2016 and 2019</summary>

</details>
<details>
<summary>List the name of all animals that are neutered and have less than 3 escape attempts</summary>

</details>
<details>
<summary>List the date of birth of all animals named either "Agumon" or "Pikachu"</summary>

</details>
<details>
<summary>List name and escape attempts of animals that weigh more than 10.5kg</summary>

</details>
<details>
<summary>Find all animals that are neutered</summary>

</details>
<details>
<summary>Find all animals not named Gabumon</summary>

</details>
<details>
<summary>Find all animals with a weight between 10.4kg and 17.3kg</summary>

</details>

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
+CREATE DATABASE vet_clinic
+
 CREATE TABLE animals (
-    name varchar(100)
-);
+    id INTEGER,
```
Comment (Ismailco) [inline schema.sql:6]: - [ ] [OPTIONAL] You can make the id column not null and autoincremented
e.g
```sql
  id int NOT NULL IDENTITY(1,1)
```

Comment (Ismailco) [inline schema.sql:6]: - [ ] [OPTIONAL] You could make the `id` column as primary key
```sql
PRIMARY KEY (id)
```

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
```
Comment (Ismailco) [inline queries.sql]: - [ ] This query should only list the `name of all animals` born between 2016 and 2019, kindly make sure to select only `name`s not all the data (`*`)

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+SELECT * FROM animals WHERE neutered IS true AND escape_attempts < 3;
```
Comment (Ismailco) [inline queries.sql]: - [ ] Same here kindly make sure to list only the `names` of the animals not all the data (`*`)

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @keebot-png, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by keebot-png** [comment]
Thank you for this honest and thorough review @Ismailco. I have corrected my mistakes and updated the screen shots in the PR description.

File: README.md [hunk 4]
```diff
@@ -206,19 +198,11 @@ Example:
```
Comment (HAMMAS-SALEEM) [inline README.md:198]: **[Optional]**

- [ ] Kindly add **Key Features** and **Future Feature** section are still empty :basket: Kindly add fill these sections to make your readme descriptive :+1:

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

### Hey @keebot-png 👋

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Requirements
✔️ Following Linters

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**